CONTRIBUTING.md

```diff
@@ -57,3 +57,15 @@ You can then repeat this process as often as you want - i.e. make more changes i
 copy to your local Maven repo, and test it in the project you chose. 
 
 For a real example, see the "local-testing-project" project in the examples directory.
+
+## Automated Testing
```

Comment:
This is good, and this file really needs updating for instructions on testing all 3 subprojects. If you're interested, can make a JIRA ticket for that for a future PR.